{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/helpers.py

```diff
@@ -1,5 +1,48 @@
-from django.contrib.staticfiles.templatetags.staticfiles import static
-from jingo import register
+import datetime
+import urllib
+import urlparse
 
+import jinja2
+from django.conf import settings as django_settings
+from django.utils.encoding import smart_str
 
-static = register.function(static)
+
+def settings(request):
+    """
+    Adds static-related context variables to the context.
+
+    """
+    return {'settings': django_settings}
+
+
+def thisyear():
+    """The current year."""
+    return jinja2.Markup(datetime.date.today().year)
+
+
+def urlparams(url_, hash=None, **query):
+    """Add a fragment and/or query paramaters to a URL.
+    New query params will be appended to exising parameters, except duplicate
+    names, which will be replaced.
+    """
+    url = urlparse.urlparse(url_)
+    fragment = hash if hash is not None else url.fragment
+
+    # Use dict(parse_qsl) so we don't get lists of values.
+    q = url.query
+    query_dict = dict(urlparse.parse_qsl(smart_str(q))) if q else {}
+    query_dict.update((k, v) for k, v in query.items())
+
+    query_string = _urlencode([(k, v) for k, v in query_dict.items()
```

Comment:
On the other hand `urlencode` cannot handle unicode so it needs `start_str` which explains the existance of `_urlencode`. I switched to `django.utils.http.urlencode` which plays well with unicode.